### PR TITLE
Document README files

### LINKED ISSUE #246: Document README.md for gems
At the moment it has the typical files with the `TODO: Delete this and the text above, and describe your gem`. 

We should modify that

https://github.com/decidim/decidim-bulletin-board/blob/b6519e6bd01037ece99083432bee06851afdfaa5/bulletin_board/ruby-client/README.md
https://github.com/decidim/decidim-bulletin-board/blob/b6519e6bd01037ece99083432bee06851afdfaa5/voting_schemes/electionguard/ruby-adapter/README.md
https://github.com/decidim/decidim-bulletin-board/blob/b6519e6bd01037ece99083432bee06851afdfaa5/voting_schemes/dummy/ruby-adapter/README.md

### PR DESCRIPTION
**Pull request by microstudi**
Document readmes for the included gems.

fixes #246

### CODE REVIEW

File: bulletin_board/ruby-client/README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/decidim/bulletin_board`. To experiment with that code, run `bin/console` for an interactive prompt.
+The Bulletin Board is a service composed by an Encryption Engine and an Append-Only Log. External queries to the Encryption Engine are allowed through a GraphQL API (see https://github.com/decidim/decidim-bulletin-board).
+
+This gem provides a collection of commands to interact with such API using Ruby. Mainly for it's use in a backend service.
 
-TODO: Delete this and the text above, and describe your gem
 
 ## Installation
 
 Add this line to your application's Gemfile:
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
Run this command to add the gem to your Gemfile:
```

File: bulletin_board/ruby-client/README.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
 ```ruby
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```console
```

File: bulletin_board/ruby-client/README.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-gem 'decidim-bulletin_board'
+gem 'decidim-bulletin_board', "~> 0.24.4"
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
bundle add decidim-bulletin_board
```

File: bulletin_board/ruby-client/README.md [hunk 4]
```diff
@@ -22,13 +23,32 @@ Or install it yourself as:
...
+
+# Perform operations
+bulletin_board.cast_vote(election_id, voter_id, encrypted_data) do |message_id|
+  # Do something else with the message_id
+end
+```
+
+See a list of available operations here: https://github.com/decidim/decidim-bulletin-board/blob/develop/bulletin_board/ruby-client/lib/decidim/bulletin_board/client.rb
+
+
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```

File: bulletin_board/server/README.md [hunk 5]
```diff
@@ -9,41 +9,57 @@ The Bulletin Board is a service composed by an Encryption Engine and an Append-O
 - System dependencies:
 - Python gmpy2 package dependencies: libgmp, libmpfr and libmpc
 
-The Bulletin Board depends on the existence of a Decidim installation.
+The Bulletin Board is intended to use alongside a Decidim installation. It's main purpose is to provide a GraphQL interface to a secure, auditable and verifiable voting system. This is currently achieved by using the [ElectionGuard python implementation](https://github.com/microsoft/electionguard-python), developed by Microsoft. So this server acts as a [wrapper to the ElectionGuard python code](../../voting_schemes/electionguard/python-wrapper).
```
Comment (andreslucena) [inline bulletin_board/server/README.md]:
```suggestion
The Bulletin Board is intended to use alongside a Decidim installation. Its main purpose is to provide a GraphQL interface to a secure, auditable and verifiable voting system. This is currently achieved by using the [ElectionGuard python implementation](https://github.com/microsoft/electionguard-python), developed by Microsoft. So this server acts as a [wrapper to the ElectionGuard python code](../../voting_schemes/electionguard/python-wrapper).
```

**Review by andreslucena — CHANGES_REQUESTED** [review]
Great improvement, I only have a few minor details and this is ready

**Comment by microstudi** [comment]
Changes applied @andreslucena

File: bulletin_board/ruby-client/README.md [hunk 6]
```diff
@@ -1,34 +1,56 @@
...
-Add this line to your application's Gemfile:
+Run this command to add the gem to your Gemfile:
 
-```ruby
-gem 'decidim-bulletin_board'
+```console
+bundle add decidim-bulletin_board
 ```
 
 And then execute:
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```

File: bulletin_board/ruby-client/README.md [hunk 7]
```diff
@@ -1,34 +1,56 @@
...
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```

File: bulletin_board/ruby-client/README.md [hunk 8]
```diff
@@ -1,34 +1,56 @@
...
 
-    $ bundle install
+```console
+bundle install
+```
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```

File: bulletin_board/ruby-client/README.md [hunk 9]
```diff
@@ -1,34 +1,56 @@
...
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
```

File: bulletin_board/ruby-client/README.md [hunk 10]
```diff
@@ -1,34 +1,56 @@
 # Decidim::BulletinBoard
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/decidim/bulletin_board`. To experiment with that code, run `bin/console` for an interactive prompt.
+The Bulletin Board is a service composed by an Encryption Engine and an Append-Only Log. External queries to the Encryption Engine are allowed through a GraphQL API (see https://github.com/decidim/decidim-bulletin-board).
 
-TODO: Delete this and the text above, and describe your gem
+This gem provides a collection of commands to interact with such API using Ruby. Mainly for it's use in a backend service.
```
Comment (andreslucena) [inline bulletin_board/ruby-client/README.md]:
```suggestion
This gem provides a collection of commands to interact with such API using Ruby, mainly for its use in a backend service.
```

File: bulletin_board/ruby-client/README.md (hunk 6 above)
Comment (andreslucena) [reply bulletin_board/ruby-client/README.md]: No need to run `bundle install` if we're doing `bundle add `

**Review by andreslucena — CHANGES_REQUESTED** [review]
A couple more details and we're good

**Review by andreslucena — APPROVED** [review]

**Comment by andreslucena** [comment]
There's a failing spec for a linter but that's already happening in the `develop` branch, so I'm merging this

**Pull request merged**